tests/framework/db/QueryBuilderTest.php

```diff
@@ -1633,6 +1633,7 @@ public function testComplexSelect()
         $query = (new Query())
             ->select([
                 'ID' => 't.id',
+                'numeric' => '1.3',
```

Comment:
This seems to be incorrect. Array values should be column names, not values - this creates inconsistency how this array is interpreted. Not mention that `1.3` may be valid column name. 

If you want something like `1.3 as numeric` you should use `Expression`.